lib/membrane_rtc_engine/metrics.ex

```diff
@@ -2,47 +2,20 @@ defmodule Membrane.RTC.Engine.Metrics do
   @moduledoc """
   Defines list of metrics, that Reporter instance can aggregate by listening on events emitted in RTC Engine.
   Suggested Reporter implementation is `Membrane.TelemetryMetrics.Reporter` from `membrane_telemetry_metrics`.
-  `Membrane.TelemetryMetrics.Reporter` started with metrics returned by `metrics/1` function will be able to generate reports, that matches type `Membrane.RTC.Engine.Metrics.rtc_engine_report()`
-  You can see usage example in (`membrane_videoroom`)[github.com/membraneframework/membrane_videoroom].
-  """
-
-  alias Membrane.RTC.Engine.Endpoint.WebRTC.TrackReceiver
-  alias Membrane.RTC.Engine.Track
+  `Membrane.TelemetryMetrics.Reporter` started with metrics returned by `metrics/1` function will be able to 
+  generate reports, that matches type `Membrane.RTC.Engine.Metrics.rtc_engine_report()`
```

Comment:
1. The types to be auto-linked should be written with `t:`
2. Reports are plural
3. The type mentioned here is now gone
```suggestion
  generate reports that match type `t:Membrane.RTC.Engine.Metrics.rtc_engine_report/0`
```